### ISSUE TITLE
Example for Writing to VarLenArray

### ISSUE THREAD

**Issue by mxkramer** [aldanor/hdf5-rust#85]
It would be very helpful to have a example for this. I can't seem to figure out how to write a 1D array of `Vec` to a dataset. Would be happy to write the example if someone could tell me how to do the writing.

**Comment by aldanor** [comment]
You want to write a 1-D array of var-len arrays?

**Comment by hombit** [comment]
I'd like to see the same thing. I have a structure like
```rust
struct Item {
    id: u64,
    size: usize,
    x: vec<f32>,
    y: vec<f32>,
}
```

Can I write a bunch of `Item`'s instances to a dataset as 1-d array?

**Comment by aldanor** [comment]
@hombit No, because HDF5 has no idea what "Vec" means. It has to be a memory layout that it can understand - e.g. if you use `VarLenArray`.
